macros/sfdc_formula_view.sql

```diff
@@ -1,4 +1,4 @@
-{%- macro sfdc_formula_view(source_table, source_name='salesforce', reserved_table_name=source_table, fields_to_include=none, full_statement_version=true, materialization='view', using_quoted_identifiers=False) -%}
+{%- macro sfdc_formula_view(source_table, source_name='salesforce', reserved_table_name=source_table, fields_to_include=none, materialization='view', using_quoted_identifiers=False, full_statement_version=true) -%}
```

Comment:
It looks like with this update, we also no longer us the following parameters anymore:
- `reserved_table_name`
- `fields_to_include`

Can you confirm these parameters are not used? If so, we should probably document this somewhere or include some warning messages to raise in the logs that these parameters are not being used and should be removed by the end user.

Reply:
Agree the parameters are not used. Removed and also mention them in the error.